app/models/workspace.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class Workspace < ApplicationRecord
+  has_many :tickets, dependent: :destroy
+  has_many :folders, dependent: :destroy
+  has_many :items, dependent: :destroy
```

Comment:
Maybe destroy `tickets` and `folders` relations and create 2 methods for Item: `tickets` and `folders`